lighthouse-cli/index.js

```diff
@@ -91,6 +96,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('mobile', true)
   .default('output', Printer.OUTPUT_MODE.pretty)
   .default('output-path', 'stdout')
+  .default('launch-chrome', true)
```

Comment:
should be `.default('skip-autolaunch', false)`? (or drop the line completely since we can rely on `undefined` being falsy